packages/lesswrong/lib/react-headroom/index.tsx

```diff
@@ -289,14 +316,14 @@ export default class Headroom extends Component {
 
     const { style, wrapperStyle, ...rest } = divProps
 
-    let innerStyle = {
+    let innerStyle: CSSProperties = {
       position: this.props.disable || this.state.state === 'unfixed' ? 'relative' : 'fixed',
       top: 0,
       left: 0,
       right: 0,
       zIndex: 1,
       WebkitTransform: `translate3D(0, ${this.state.translateY}, 0)`,
-      MsTransform: `translate3D(0, ${this.state.translateY}, 0)`,
+      msTransform: `translate3D(0, ${this.state.translateY}, 0)`,
```

Comment:
`csstype` complained about the capitalization here.  See this [thread](https://github.com/facebook/react/issues/2950) for more context.